client/src/components/layouts/Certification.js

```diff
@@ -22,8 +23,22 @@ class CertificationLayout extends Component {
     }
     this.props.executeGA({ type: 'page', data: pathname });
   }
+
   render() {
-    return <Fragment>{this.props.children}</Fragment>;
+    const { children, theme = 'default', useTheme = true } = this.props;
+
+    return (
+      <Fragment>
+        <Helmet
+          bodyAttributes={{
+            class: useTheme
+              ? `${theme === 'default' ? 'light-palette' : 'dark-palette'}`
+              : 'light-palette'
+          }}
+        ></Helmet>
+        {children}
+      </Fragment>
+    );
```

Comment:
I don't think we can theme the certificates, because it messes up their css.  Also, (because of Gatsby, I assume) th theme is currently being ignored, at least when I try it locally.

Reply:
If I remember correctly, it was intentionally decided that the lansing page and the certs will always default to light mode.